test/storage/page_guard_test.cpp

```diff
@@ -22,32 +22,175 @@
 
 namespace bustub {
 
-// NOLINTNEXTLINE
-TEST(PageGuardTest, DISABLED_SampleTest) {
-  const size_t buffer_pool_size = 5;
-  const size_t k = 2;
+const size_t FRAMES = 10;
```

Comment:
Consider renaming this? This is actually still size, isn't it?

Reply:
# of buffer frames is more accurate.